ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -70,15 +70,28 @@ def make_demo_buffer(
         for i, obs in enumerate(split_obs.visual_observations):
             demo_raw_buffer["visual_obs%d" % i].append(obs)
         demo_raw_buffer["vector_obs"].append(split_obs.vector_observations)
-        # TODO: update the demonstraction files and read from the new proto format
-        if behavior_spec.action_spec.continuous_size > 0:
-            demo_raw_buffer["continuous_action"].append(
-                current_pair_info.action_info.vector_actions_deprecated
-            )
-        if behavior_spec.action_spec.discrete_size > 0:
-            demo_raw_buffer["discrete_action"].append(
-                current_pair_info.action_info.vector_actions_deprecated
-            )
+        if (
+            current_pair_info.action_info.continuous_actions is None
+            and current_pair_info.action_info.continuous_actions is None
```

Comment:
```suggestion
            and current_pair_info.action_info.discrete_actions is None
```
(I think, not totally sure)